web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -236,26 +236,26 @@ export const formatters: Formatters = {
     type: 'exec',
     desc: 'Command Execution',
     format: event => {
-      const { proto, kubernetes_cluster, user = '' } = event;
+      const { proto, kubernetes_cluster, user = '', sid } = event;
       if (proto === 'kube') {
         if (!kubernetes_cluster) {
-          return `User [${user}] executed a Kubernetes command`;
+          return `User [${user}] executed a Kubernetes command within a session [${sid}]`;
```

Comment:
Here and below, let's remove `a`.
```suggestion
          return `User [${user}] executed a Kubernetes command within session [${sid}]`;
```

Reply:
Thanks @zmb3, updated to that language.